Resources/Prototypes/_Harmony/Catalog/Fills/Crates/armory.yml

```
@@ -44,3 +44,27 @@
       - id: SpeedLoaderLightRifle
         orGroup: ammo
         amount: 42
+
+- type: entity
+  id: CrateArmorySMGHarmony
+  suffix: Drozd
+  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
+  name: SMG crate
+  description: Contains two high-powered, automatic rifles with four mags. Requires Armory access to open.
```

The order is incorrect, it should be type > abstract > parent > id > categories > name > suffix > description > components
```suggestion
  parent: [ CrateWeaponSecure, BaseSecurityContraband ]
  id: CrateArmorySMGHarmony
  name: SMG crate
  suffix: Drozd, Secure
  description: Contains two high-powered, automatic rifles with four mags. Requires Armory access to open.
```